schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+	id INT,
+	name TEXT NOT NULL,
```

Comment:
- [ ] Kindly I recommend you to make the data type of name column `varchar` with their max-length Just like this :- 
`name varchar(255)`, instead of datatype `TEXT`.